cosmwasm/register-p2p-tokens.ts

```diff
@@ -0,0 +1,219 @@
+    const interchainTokenServiceAddress = config.getContractConfig('InterchainTokenService').address;
+
+    if (!interchainTokenServiceAddress) {
+        throw new Error('InterchainTokenService contract address not found');
+    }
+
+    await forEachTokenInFile(config, options, async (tokenData: SquidToken, tokenOnChain: SquidTokenData) => {
+        try {
+            const registered = await checkSingleTokenRegistration(
+                client,
+                interchainTokenServiceAddress,
+                tokenData.tokenId,
+                tokenOnChain.axelarChainId,
+            );
+            tokenOnChain.registered = registered ? true : false;
+            printInfo(`Token ${tokenData.tokenId} on ${tokenOnChain.axelarChainId} is ${registered ? 'registered' : 'not registered'}`);
+        } catch (e) {
+            printError(`Error checking token ${tokenData.tokenId} on ${tokenOnChain.axelarChainId}: ${e}`);
+        }
+    });
+}
+
+const programHandler = () => {
+    const program = new Command();
+
+    program
+        .name('ITS p2p token migration script')
+        .version('1.0.0')
+        .description(
+            'The script will register the P2P tokens to the ITS Hub or check if they are already registered.\n' +
+                'Requires the token file to be present in the following path:\n' +
+                ' * `axelar-chains-config/info/tokens-p2p/tokens-${env}.json`\n' +
+                'The tokens file should follow the Squid config format.\n',
+        );
+
+    program
+        .command('register-tokens')
+        .description('Register tokens to the ITS Hub.')
+        .addOption(new Option('-chains, --chains <chains...>', 'chains to run the script for. Default: all chains').env('CHAINS'))
+        .addOption(
+            new Option('-tokenIds, --tokenIds <tokenIds...>', 'tokenIds to run the script for. Default: all tokens').env('TOKEN_IDS'),
+        )
+        .addOption(new Option('-dryRun, --dryRun', 'provide to just print out what will happen when running the command.'))
+        .addOption(new Option('-env, --env <env>', 'environment to run the script for').env('ENV').makeOptionMandatory(true))
+        .addOption(
+            new Option('-m, --mnemonic <mnemonic>', 'Mnemonic of the InterchainTokenService operator account')
+                .makeOptionMandatory(true)
+                .env('MNEMONIC'),
+        )
+        .action((options) => {
+            mainProcessor(registerTokensInFile, options, []);
+        });
+
+    program
+        .command('check-tokens')
+        .description('Check tokens registration status on the ITS Hub.')
+        .addOption(new Option('-chains, --chains <chains...>', 'chains to run the script for. Default: all chains').env('CHAINS'))
+        .addOption(
+            new Option('-tokenIds, --tokenIds <tokenIds...>', 'tokenIds to run the script for. Default: all tokens').env('TOKEN_IDS'),
```

Comment:
```suggestion
            new Option('--tokenIds <tokenIds...>', 'tokenIds to run the script for. Default: all tokens').env('TOKEN_IDS'),
```
Single `-` flag is usually used for shorthands, e.g `-n`. It's also not always needed, so you can remove it and keep things consistent. Same comment everywhere